source/usage-examples/deleteOne.txt

```diff
@@ -3,3 +3,51 @@ Delete a Document
 =================
 
 .. default-domain:: mongodb
+
+You can delete a single document in a collection using the
+``DeleteOne()`` method.
+
+The following example specifies a query filter that matches documents in
+the ``movies`` collection with the value "Twilight" in the ``title`` field, and
+deletes the first document that matches. 
+
+.. include:: /includes/quick-start/run-example-tip.rst
+
+.. literalinclude:: /includes/quick-start/deleteOne.go
+   :start-after: begin deleteOne
+   :end-before: end deleteOne
+   :emphasize-lines: 2
+   :language: go
+   :dedent:
+
+Click here <TODO> to see a fully runnable example.
+
+Expected Result
+---------------
+
+You should not be able to find the following document: 
+
+.. code-block:: json
+
+    { "_id": { "$oid": "573a13bff29313caabd5e06b" }, 
+      ...
+      "title": "Twilight",
+      ...
+    }
+
+To check if you successfully deleted this document, try performing a
+find operation with the ``title`` field on the collection. For an
+example of how to perform the find operation, see our <TODO: Find One
+Usage Example guide>. 
```

Comment:
```suggestion
For an example of finding a document, see our <TODO: Find One
Usage Example>. 
```
Suggestion: I think this can be shortened.

Suggestion: I think the clause 

> "try performing a find operation with the ``title`` field on the collection"

is imprecise, and should either be deleted or made more precise.

Reply:
I made this more percise.

Reply:
Zoom call. Agree the new wording should replace the intro sentence for the result.